### PR TITLE
Sema: Fix bogus 'parameters may not have the 'var' specifier' diagnostic

### PR DESCRIPTION
**Pull request by slavapestov**
Small regression with Swift 3 language changes.

### CODE REVIEW

**Comment by slavapestov** [comment]
@swift-ci Please smoke test

File: include/swift/AST/Decl.h [hunk 1]
```diff
@@ -288,6 +288,11 @@ class alignas(1 << DeclAlignInBits) Decl {
     /// once (either in its declaration, or once later), making it immutable.
     unsigned IsLet : 1;
 
+    /// \brief Whether this is an 'inout' parameter; this is preferrable
```
Comment (xwu) [inline include/swift/AST/Decl.h]: Spelling: "preferable"

Comment (slavapestov) [inline include/swift/AST/Decl.h]: Thanks :)

**Comment by slavapestov** [comment]
@swift-ci Please smoke test and merge

**Pull request merged**